css/style.css

```diff
@@ -1,61 +1,62 @@
 * {
-	font-family: 'Open Sans', 'Helvetica Neue', sans-serif;
-	margin: 0;
+  font-family: 'Open Sans', 'Helvetica Neue', sans-serif;
+  margin: 0;
 }
 
 .container-fluid {
-	display: flex;
-	flex-direction: row;
```

Comment:
I think it would be better to eliminate "flex-direction: row", because "display: flex" by default makes it a row. 👍

Reply:
Great job with your review.
Please when applying reviews it's recommended to be more gentle and polite. Also, its recommended to always provide a reason (`why`) for the review